OpenDF-Describe-Image/src/Server/server.py

```diff
@@ -23,16 +22,19 @@
 app = Flask(__name__)
 app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
 
+#images dictionary contains uuids and corresponding file path for each uuid
```

Comment:
This will be a temporary array, just to support `get_actual_path(uuid):` function. Once completed, this function will fetch these details from Elastisearch.